dagre/dagre.d.ts

```diff
@@ -10,20 +10,24 @@ declare module Dagre{
 
     interface Graph {
         new (): Graph;
-        edges(): string[];
-        edge(id: string): any;
+        edges(): Edge[];
+        edge(id: any): any;
```

Comment:
Because it turns out that you can also input an object in to edge() to get an edge returned. In certain cases, that's the only way to access some edge details.